LayoutTests/fast/forms/color/display-none-input-color-chooser-shown.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
You need -expected.txt.

Reply:
Yeah! I missed it.. Shouldn't push patches late night.. Also do you have any idea for what alternative we have for "testRunner.isChooserShown", because this test case does not work 1:1 in Webkit. I tried to look into folders to find something related but couldn't. Would appreciate your help. Thanks!